app/test_main.py

```diff
@@ -2,0 +1,28 @@
+from unittest import mock
+from unittest.mock import MagicMock
+
+from app.main import can_access_google_page
+
+
+url = "https://google.com"
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_access_google_page(
+        mocked_url: MagicMock,
+        mocked_connection: MagicMock
+) -> None:
+    possible_variants = [
+        [True, True],
+        [True, False],
+        [False, True],
+        [False, False]
+    ]
+    for variant in possible_variants:
+        mocked_url.return_value, mocked_connection.return_value = variant
+        print(mocked_url.return_value, mocked_connection.return_value)
```

Comment:
Remove the `print` statement used for debugging. According to the checklist, you should delete prints and other debugging code before committing your solution.